fast/stages/0-bootstrap/organization.tf

```diff
@@ -266,6 +291,9 @@ module "organization" {
       description = "Organization policy conditions."
       iam         = {}
       values = merge(
+        {
+          allowed-essential-contacts-domains-all = {}
+        },
```

Comment:
nit: no need for an extra map, just declare it as an extra key in the one containing the DRS tag value

Reply:
<3